src/openclsim/plot/log_dataframe.py

```diff
@@ -2,11 +2,26 @@
 
 import pandas as pd
 
+from openclsim.model import get_subprocesses
 
-def get_log_dataframe(simulation_object, activities=[]):
-    """Get the log of the simulation objects in a pandas dataframe."""
+def get_log_dataframe(simulation_object, id_map=[]):
```

Comment:
use `id_map=None` and instantiate in function call. Now activities is shared among all instances. This can lead to difficult to find [bugs](https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument).